internal/handlers/dnd5e/v1alpha1/handler.go

```diff
@@ -1241,6 +1241,22 @@ func convertCharacterToProto(char *dnd5e.Character) *dnd5ev1alpha1.Character {
 			Wisdom:       char.AbilityScores.Wisdom,
 			Charisma:     char.AbilityScores.Charisma,
 		},
+		AbilityModifiers: &dnd5ev1alpha1.AbilityModifiers{
```

Comment:
calculating modifiers does not belong in the handler. handlers are converters only